.github/workflows/e2e-tests.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         wp_version: ['latest', 'nightly']
         include:
           - amp_version: '1.5.5'
-            wp_version: '5.2.16'
+            wp_version: '5.2.21'
```

Comment:
This is the latest point version which should really be our true minimum supported version. It was only updated here to invoke a run of the workflow but it can be kept as a worthwhile update but unrelated to the fix.